ctl/src/cli.rs

```diff
@@ -107,6 +109,8 @@ pub enum ApplicationCmd {
     send_proxy: bool,
     #[structopt(long = "expect-proxy", help = "Configures the client-facing connection to receive a PROXY protocol header version 2")]
     expect_proxy: bool,
+    #[structopt(long = "lb-policy", help = "Configures the load balacing policy")]
```

Comment:
"balacing" -> "balancing"
please grep the source for other instances of the "balacing" word
also, please rename "lb-policy" to "load-balancing-policy", then maybe add a "lb" shortcut if needed

Reply:
please rename the command line options here